dlt/destinations/impl/qdrant/factory.py

```diff
@@ -26,6 +28,20 @@ def _raw_capabilities(self) -> DestinationCapabilitiesContext:
 
         return caps
 
+    @classmethod
+    def adjust_capabilities(
+        cls,
+        caps: DestinationCapabilitiesContext,
+        config: QdrantClientConfiguration,
+        naming: t.Optional[NamingConvention],
+    ) -> DestinationCapabilitiesContext:
+        caps = super(qdrant, cls).adjust_capabilities(caps, config, naming)
+        if config.credentials.is_local():
+            # Local qdrant can not load in parallel
+            caps.loader_parallelism_strategy = "sequential"
+            caps.max_parallel_load_jobs = 1
```

Comment:
workaround for local qdrant, it is not thread safe

Reply:
very true! this is why this capability got added :)